controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -296,6 +296,23 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 				return reconcile.Result{}, err
 			}
 
+			//TODO: Verify the caps
```

Comment:
Why do we need this comment?

Reply:
We need to verify the caps, as these caps are similar to cephFs and might provide access to multiple radosnamesapces

Reply:
But why a comment in the code? if we don't need to change then we need to create another PR just to remove the comment. Please capture this somewhere else? (an issue, an internal todo list, etc.)

Reply:
ack, updating it